### PR TITLE
Adding Ozair's profile

### PR DESCRIPTION
**Pull request by ozairashfaqueSSW**

### CODE REVIEW

File: Ozair-Ashfaque/Ozair-Ashfaque.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+id: "c5218995-5559-ec11-8f8f-00224815a714"
```
Comment (pierssinclairssw) [inline Ozair-Ashfaque/Ozair-Ashfaque.md:2]: Needs to match CRM, aka c752950a-2761-ec11-8f8f-002248181149

File: Ozair-Ashfaque/Ozair-Ashfaque.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+quoteAuthor: "Bill Gates"
+
+---
+
+
+
+Ozair is a Web Developer working at SSW Melbourne with an exprience in web development building responsive and scaleable websites and web applications.
+
+He holds a masters degree of Network System with specialization in Cyber Security from Swinburne University of Technology.He specializes in front-end development using JavaScript frameworks such as Angular and React, and has experience in back-end development using .NET Core.
```
Comment (pierssinclairssw) [inline Ozair-Ashfaque/Ozair-Ashfaque.md:16]: change to 

He holds a masters degree of Network Systems with a specialization in Cyber Security from Swinburne University of Technology. He specializes in front-end development using JavaScript frameworks such as Angular and React, and has experience in back-end development using .NET.

File: Ozair-Ashfaque/Ozair-Ashfaque.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+quote: "I choose a lazy person to do a hard job. Because a lazy person will find an easy way to do it."
+
+quoteAuthor: "Bill Gates"
+
+---
+
+
+
+Ozair is a Web Developer working at SSW Melbourne with an exprience in web development building responsive and scaleable websites and web applications.
```
Comment (pierssinclairssw) [inline Ozair-Ashfaque/Ozair-Ashfaque.md:14]: change to

Ozair is a Web Developer working at SSW Melbourne with experience in web development building responsive and scalable websites and web applications.

**Comment by ozairashfaqueSSW** [comment]
@pierssinclairssw  Please have a review

**Review by pierssinclairssw — APPROVED** [review]

**Pull request merged**